lib/features/notes/core/converters/delta_to_pdf.dart

```diff
@@ -0,0 +1,146 @@
+/// delta_to_pdf
+/// [title] DeltaToPDF Class
+/// [Description] This is class with helper functions to convert flutter_quill delta object to pdf object
+/// [Written_by] mmjsmohit
+
+library delta_to_pdf;
+
+import 'package:pdf/pdf.dart';
+import 'package:pdf/widgets.dart' as pw;
+import 'package:flutter_quill/flutter_quill.dart' as fq;
+
+/// [title] DeltaToPDF Class
+/// [Description] This is class with helper functions to convert flutter_quill delta object to pdf object
+/// [Written_by] mmjsmohit
+class DeltaToPDF {
+  /// [Description]  converts header text style
+  /// [Returns] fontWeight & fontSize
+  getPDFHeaderStyle(key, value) {
+    pw.FontWeight fontWeight = pw.FontWeight.normal;
+    double fontSize = 14;
```

Comment:
```suggestion
    double fontSize = 14;
    
```
Please add a white line after all declaration statements.